### ISSUE TITLE
Shove

### ISSUE THREAD

**Issue by D-D-2** [Cards-of-Chaos/Game-Design#27]
Expelling wind from your hands you dash at target creature dealing 5 wind damage and they must make a spell saving throw. On failed save, the creature is moved 3 spaces and is knocked prone.

Range: 2 Spaces.

**Comment by D-D-2** [comment]
I'm seeing The Incredible Hulk scene in my head with this spell. I assume it means they are moved 3 spaces backwards in the direction you struck them at? For example, if I was 2 spaces away diagonally from the south west, They would be moved back 3 spaces north east. Yes?

**Comment by nstevens26** [comment]
Yes, exactly. I should update it to say moved backwards three spaces or three spaces in a direction of your choice

**Comment by BoshBerry** [comment]
If a creature passes the save, it is likely that you would be sharing a space with an enemy. I assume this is not intended?  One suggested fix would be to turn into a pure utility spell with slightly more movement speed and no damage: 

Move up to 3 spaces to an unoccupied space in one direction. All creatures in your path make a power saving throw against your chaos DC. On a failed save, each creature is moved 3 spaces in the direction of travel and knocked prone.
